assume/strategies/learning_advanced_orders.py

```diff
@@ -582,11 +596,13 @@ def load_actor_params(self, load_path):
 
         params = th.load(directory, map_location=self.device)
 
-        self.actor = Actor(self.obs_dim, self.act_dim, self.float_type)
+        self.actor = self.policy_class(self.obs_dim, self.act_dim, self.float_type)
         self.actor.load_state_dict(params["actor"])
```

Comment:
@kim-mskw please check if that's necessary. I think "unique_obs_dim" and "num_timeseries_obs_dim" would be needed for LSTM with advanced orders. Therefore copied the actor instantiation from TD3's create_actors() function. ".to(self.device)" might not be needed?

Reply:
yes it makes sense to have these here now!
The .to(self.deivce) is used to move a model or tensor to a specified device, which can be either a CPU or a GPU. Why would it not be needed here?

Reply:
'.to(self.device)' wasn't used before and I'm not sure if line 597 'params = th.load(directory, map_location=self.device)' therefore already covers this part.

Reply:
Ah understood, from my perspective it is however necesary that the actor model itself is on the device and not only the params.

Reply:
We also use it for the critics.